subsys/zigbee/lib/zigbee_app_utils/zigbee_app_utils.c

```diff
@@ -856,3 +868,47 @@ void zigbee_configure_sleepy_behavior(bool enable)
 	}
 }
 #endif
+
+/* Wrapper allowing to schedule zb_bdb_reset_via_local_action as an alarm */
+static void factory_reset(zb_uint8_t arg)
```

Comment:
As a general rule - the ZBOSS callback prototype uses `zb_bufid_t` as the argument type. The `arg` name is fine. It can be also `bufid` or even `unused`,

Reply:
@tomchy Thanks for explanation, I will correct it.